### PR TITLE
addaws to sponsor list

### PR DESCRIPTION
**Pull request by lovepoem**

### CODE REVIEW

**Comment by zhaofeng-shu33** [comment]
need also change some js code.

**Comment by zhaofeng-shu33** [comment]
no template for `sponsor` key in `src/pages/hackathon/index.jsx` yet.

File: src/pages/hackathon/index.jsx [hunk 1]
```diff
@@ -200,7 +201,14 @@ class Hackathon extends Language {
 				<div><a href="https://www.linkstars.com/"><img src="/images/hackathon/holder_coorperation/holder_coorperation3.png" /></a></div>
 				<div><a href="https://www.agora.io/"><img src="/images/hackathon/holder_coorperation/holder_coorperation4.png" /></a></div>
 			</div>
-
+            <div id="holder">
+                <h1>Sponsor</h1>
+				{sponsorSource.list.map((item, index) => (
+					<div key={index}>
+                        <a href={item.link}><img src={"/images/hackathon/" + item.image} />
```
Comment (lovepoem) [inline src/pages/hackathon/index.jsx:208]: 不用加的，每一页的页面底部都有了

Comment (zhaofeng-shu33) [reply src/pages/hackathon/index.jsx:208]: Originally the sponsor is scattered within `community` part of Hackathon. I think we should make it a separate part.

**Review by zhaofeng-shu33 — APPROVED** [review]

**Pull request merged**